pvanalytics/tests/quality/test_data_shifts.py

```diff
@@ -0,0 +1,102 @@
+"""Tests for data shift quality control functions."""
+import pandas as pd
+import pytest
+from pvanalytics.quality import data_shifts as dt
+from ..conftest import DATA_DIR
+import sys
+import ruptures
+
+
+test_file_1 = DATA_DIR / "pvlib_data_shift.csv"
+
+
+@pytest.fixture
+def generate_time_series():
+    # Pull down the saved PVLib dataframe and process it
+    df = pd.read_csv(test_file_1)
+    signal_no_index = df['value']
+    df.index = pd.to_datetime(df['timestamp'])
+    signal_datetime_index = df['value']
+    changepoint_date = df[df['label'] == 1].index[0]
+    df_weekly_resample = df.resample('W').median()['value']
+    return (signal_no_index, signal_datetime_index,
+            df_weekly_resample, changepoint_date)
+
+
+def requires_ruptures(test):
```

Comment:
There's a copy of this in `test_time.py`; might as well move it to `conftest.py` next to `requires_pvlib` and import it into the two test files instead of duplicating.